### PR TITLE
Views: Layer List Container Buildout

### PR DESCRIPTION
**Pull request by gblakeman**
Builds out the item container and bottom tray components. A few notes:

* The tray is functional (opens/closes) and responds to selections on the Figma dartboard, but in-list selection handling is coming in a followup PR.
* I’m still using the original `RealishViewObject` view type which includes a bunch of stuff that’s no longer needed (and is quite slow). This will get sorted when I get into the backend work for the plugin.
* Removes some legacy views stuff that’s no longer necessary.

----

### CODE REVIEW

File: src/views/ItemsList.svelte [hunk 1]
```diff
@@ -1,40 +1,67 @@
...
+  import ItemsListHeader from './ItemsListHeader';
 
-  export let items;
+  export let isExpanded: boolean = false;
+  export let items: Array<RealishViewObject> = [];
+
+  const setFirstItem = (
+    currentItems: Array<RealishViewObject>,
+  ): RealishViewObject => {
+    const firstIndex: number = 0;
```
Comment (joeyyang) [inline src/views/ItemsList.svelte:11]: can this just be `return currentItems[0];`?

File: src/views/stores.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import type { Writable } from 'svelte/store';
+import { writable } from 'svelte/store';
+
+// ---- writeable
```
Comment (joeyyang) [inline src/views/stores.ts:4]: according to the dictionary, both `writable` and `writeable` are accepted spellings - but wanted to flag the different spelling here

**Review by joeyyang — APPROVED** [review]
Couple small comments but otherwise looks good!

File: src/views/BottomTray.svelte [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  {/if}
+</svelte:head>
+
+<ItemsList
+  isExpanded={$isTrayOpen}
+  items={items}
+  on:handleAction={(customEvent) => runAction(customEvent.detail)}
+/>
+<FooterBar
+  hasItems={items && items.length}
```
[the remark this answers is not in the document]
Comment (gblakeman) [reply src/views/BottomTray.svelte:38]: Not in Svelte markup yet, unfortunately. 😔

File: src/views/stores.ts (hunk 2 above)
Comment (gblakeman) [reply src/views/stores.ts:4]: 🤦🏻‍♂️

File: src/assets/css/shared/_colors.scss [hunk 4]
```diff
@@ -45,6 +45,7 @@ $wl-type-data-label-placeholder: $black-40o;
 $wl-type-header-secondary: $black-60o;
 $wl-type-viewer: $blue-100o;
 $wl-type-list: $purple-100o;
+$wl-type-list-invert: $black-0o;
```
[the remark this answers is not in the document]
Comment (gblakeman) [reply src/assets/css/shared/_colors.scss:48]: Yeah, good call. ⬜️⬛️

**Pull request merged**